rebar.config

```diff
@@ -10,5 +10,7 @@
 {deps, [
         lager,
         cowboy,
-        aeserialization
+        aeserialization,
+        {jsx, {git, "https://github.com/talentdeficit/jsx.git",
+                     {tag, "2.8.0"}}}
```

Comment:
A peculiarity with the devmode plugin is that it expects you to export `AE_ROOT`, pointing to a source tree of the `aeternity` node. Any dependencies (like `jsx`) that exist in the `aeternity` source, should then just be listed by name, like the ones already in the `deps`.